smarts/env/gymnasium/wrappers/metric/metrics.py

```diff
@@ -424,17 +432,19 @@ def _check_scen(scenario: Scenario, agent_interfaces: Dict[str, AgentInterface])
         for agent_name, agent_mission in scenario.missions.items()
     }
 
+    aoi = scenario.metadata.get("actor_of_interest_re_filter", None)
     for agent_name, agent_interface in agent_interfaces.items():
         interest_criteria = agent_interface.done_criteria.interest
         if not (
-            (goal_types[agent_name] == PositionalGoal and interest_criteria is None)
+            (goal_types[agent_name] == PositionalGoal and interest_criteria == None)
```

Comment:
The standard is to use `is None` for checking against `None` because the reference and values are both equal, but either works.